packages/eds-tokens/package.json

```diff
@@ -55,7 +42,7 @@
     "elements.css"
   ],
   "scripts": {
-    "build": "rollup -c --bundleConfigAsCjs && pnpm run types && pnpm --filter @equinor/eds-tokens-sync run build && pnpm --filter @equinor/eds-tokens-build run build",
+    "build": "rollup -c --bundleConfigAsCjs && pnpm run types",
```

Comment:
I guess we can assume that the token-sync and token-build are pre-build, but I think it makes sense to add `tokens-in-code:build` here. Then the build of all the things are isolated in the package, and we don't have to specify both the old and new build in the root package.json.

Reply:
No they absoluetly should be seperate

Reply:
`tokens-in-code:build` should be considered as part of the transform step. It is only ran when we do `transform:tokens`. The reason for this is that `lightningcss` is an external dependency that will get updated over time which could affect the output. Changes to output must always be verified to make sure nothing unexpectely changes. Unlike 1.0 tokens, 2.0 tokens does not have a build step, only sync and transform